src/schema/schema.ts

```diff
@@ -410,6 +413,18 @@ export function createZimbraSchema(
 				revokeRights: (_, variables) =>
 					client.revokeRights(variables.input as RevokeRightsInput),
 				revokeTrustedDevice: client.revokeTrustedDevice,
+				setCustomMetadata: (_: any, variables: any) => client
+						.jsonRequest({
```

Comment:
we don't do jsonRequest from here directly, it needs to be moved to batch-client

Reply:
Done